rakelib/check_changelog.rb

```diff
@@ -45,6 +48,12 @@ def check_changelog
       link_url = "#{m[2]}##{m[4]}"
       { line: idx+1, message: "Link text is #{link_text} but links points to #{link_url}." }
     end)
+
+    # Flag common typos in GitHub issue/PR URLs
+    typo_links = line.scan(links_typos)
+    all_warnings.concat Array(typo_links.map do |m|
+      { line: idx+1, message: "This looks like a GitHub link URL with a typo. Issue links should use `/issues/123` (plural) and PRs links should use `/pull/123` (singular)." }
```

Comment:
```suggestion
      { line: idx+1, message: "This looks like a GitHub link URL with a typo. Issue links should use `/issues/123` (plural) and PR links should use `/pull/123` (singular)." }
```

Reply:
You mean I made a typo in the code that warns about typos? 😆

Reply:
fixed.